plugins/modules/certificate_complete_chain.py

```diff
@@ -150,6 +156,8 @@
     import cryptography.hazmat.primitives.serialization
     import cryptography.hazmat.primitives.asymmetric.rsa
     import cryptography.hazmat.primitives.asymmetric.ec
+    import cryptography.hazmat.primitives.asymmetric.ed448
+    import cryptography.hazmat.primitives.asymmetric.ed25519
```

Comment:
These two imports must not be there, otherwise this won't work with older cryptography versions:
```suggestion
```

Reply:
I've got some testing to do then.

I thought I was getting errors without them, but the most recent run seems to be working okay without them.  I want to do further testing.

Reply:
Well, I could never reproduce whatever bug caused me to add them in the first place.  It works without them.  Bug may have been elsewhere and I didn't diagnose it correctly.

Reply:
The imports are only necessary if you want to make absolutely sure that these imports are available. But since these are optional features (checked with the feature flags - which are obtained by testing imports in plugins/module_utils/crypto/basic.py) there's no need to do the imports here, and in fact it breaks the module for older cryptography versions which don't have these imports.